internal/flink/command_detached_savepoint.go

```diff
@@ -0,0 +1,66 @@
+package flink
+
+import (
+	"github.com/spf13/cobra"
+
+	cmfsdk "github.com/confluentinc/cmf-sdk-go/v1"
+
+	pcmd "github.com/confluentinc/cli/v4/pkg/cmd"
+)
+
+type detachedSavepointOut struct {
+	Name              string `human:"Name" serialized:"name"`
+	Path              string `human:"Path,omitempty" serialized:"path,omitempty"`
+	Format            string `human:"Format,omitempty" serialized:"format,omitempty"`
+	Limit             int32  `human:"Backoff Limit,omitempty" serialized:"backoff_limit,omitempty"`
+	CreationTimestamp string `human:"Creation Timestamp,omitempty" serialized:"creation_timestamp,omitempty"`
+	Uid               string `human:"Uid,omitempty" serialized:"uid,omitempty"`
```

Comment:
```suggestion
	Uid               string `human:"UID,omitempty" serialized:"uid,omitempty"`
```

Acronyms like this should be capitalized for human readable output